SimTKmath/Optimizers/src/IpOpt/IpTNLPAdapter.cpp

```diff
@@ -292,7 +292,7 @@ namespace SimTKIpopt
           }
         }
         else if (lower_bound > upper_bound) {
-          char string[128];
+          char string[129];
```

Comment:
@sherm1 this change gets rid of a GCC7 compiler error (#694). Are you okay with this change (and the one below)?